huaweicloud/resource_huaweicloud_sfs_turbo.go

```diff
@@ -195,13 +239,41 @@ func buildTurboCreateOpts(cfg *config.Config, d *schema.ResourceData) shares.Cre
 	return result
 }
 
+func validateParameter(d *schema.ResourceData) error {
+	_, okHpcBandwidth := d.GetOk("hpc_bandwidth")
+	_, okHpcCacheBandwidth := d.GetOk("hpc_cache_bandwidth")
```

Comment:
```suggestion
	_, isHpcBandwidthSet := d.GetOk("hpc_bandwidth")
	_, isHpcCacheBandwidthSet := d.GetOk("hpc_cache_bandwidth")
```
okXXX is hard to understand, think about isXXXSet or hasXXXSet

Reply:
OK